pkgs/by-name/la/lasuite-docs/package.nix

```diff
@@ -0,0 +1,114 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+let
+  python = python3.override {
+    self = python3;
+    packageOverrides = self: super: {
+      django = super.django_5;
+    };
+  };
+in
+
+python.pkgs.buildPythonApplication rec {
+  pname = "lasuite-docs";
+  version = "3.1.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "suitenumerique";
+    repo = "docs";
+    tag = "v${version}";
+    hash = "sha256-QeHtQWeCk1jwwtbPgQVtTKGIOUnsg9uERgrYNDl8xRk=";
+  };
+
+  sourceRoot = "source/src/backend";
+
+  build-system = with python.pkgs; [ setuptools ];
+
+  dependencies = with python.pkgs; [
+    beautifulsoup4
+    boto3
+    celery
+    django
+    django-configurations
+    django-cors-headers
+    django-countries
+    django-extensions
+    django-filter
+    django-parler
+    django-redis
+    django-storages
+    django-timezone-field
+    django-treebeard
+    djangorestframework
+    drf-spectacular
+    drf-spectacular-sidecar
+    dockerflow
+    easy-thumbnails
+    factory-boy
+    gunicorn
+    jsonschema
+    lxml
+    markdown
+    mozilla-django-oidc
+    nested-multipart-parser
+    openai
+    psycopg
+    pycrdt
+    pyjwt
+    pyopenssl
+    python-magic
+    redis
+    requests
+    sentry-sdk
+    whitenoise
+  ];
+
+  pythonRelaxDeps = true;
+
+  prePatch = ''
+    substituteInPlace impress/settings.py \
+      --replace-fail "DATA_DIR = " "DATA_DIR = os.getenv('DATA_DIR') #"
+  '';
+
+  postBuild = ''
+    export DATA_DIR=$(pwd)/data
+    ${python.pythonOnBuildForHost.interpreter} manage.py collectstatic --noinput
+  '';
+
+  installPhase =
+    let
+      pythonPath = python.pkgs.makePythonPath dependencies;
+    in
+    ''
+      runHook preInstall
+
+      mkdir -p $out/lib/docs/src
+      cp -r {core,demo,impress,locale,manage.py,__init__.py} $out/lib/docs/src
+      cp -r data/static $out/lib/docs
+      chmod +x $out/lib/docs/src/manage.py
+      makeWrapper $out/lib/docs/src/manage.py $out/bin/docs \
+        --prefix PYTHONPATH : "${pythonPath}"
+      makeWrapper ${lib.getExe python.pkgs.celery} $out/bin/celery \
+        --prefix PYTHONPATH : "${pythonPath}:$out/lib/docs/src"
+      makeWrapper ${lib.getExe python.pkgs.gunicorn} $out/bin/gunicorn \
+        --prefix PYTHONPATH : "${pythonPath}:$out/lib/docs/src"
+
+      runHook postInstall
+    '';
```

Comment:
Do we need to overwrite the complete installPhase? Is installing docs into `${python.sitePackages` a problem? Is the install incomplete otherwise? I would imagine we would just need to create the wrapper and maybe fix permissions.

Reply:
Then I will recommend looking at pretix instead, which copies manage.py in postInstall.